modules/bitgo/test/v2/unit/internal/tssUtils/ecdsa.ts

```diff
@@ -452,6 +449,23 @@ describe('TSS Ecdsa Utils:', async function () {
 
       sandbox.verifyAndRestore();
     });
+
+    async function setupSignTxRequest(isTxRequest = true) {
+      let response = { txRequests: [{ ...txRequest }] };
+      if (isTxRequest) {
+        await nockGetTxRequest({ walletId: wallet.id(), txRequestId: txRequest.txRequestId, response: response });
+      }
+      const aRecord = ECDSAMethods.convertAShare(mockAShare);
+      const signatureShares = [aRecord];
+      txRequest.signatureShares = signatureShares;
+      response = { txRequests: [{ ...txRequest }] };
+      await nockGetTxRequest({ walletId: wallet.id(), txRequestId: txRequest.txRequestId, response: response });
+      const dRecord = ECDSAMethods.convertDShare(mockDShare);
+      signatureShares.push(dRecord);
+      response = { txRequests: [{ ...txRequest }] };
+      await nockGetTxRequest({ walletId: wallet.id(), txRequestId: txRequest.txRequestId, response: response });
```

Comment:
we could use persist here to avoid repeating the same nock twice

Reply:
they are not the same, the responses are different

Reply:
But seems like the response in line 465 is reused in the nocks in lines 466 and 467?

Reply:
that part is the same, so nock can handle the response one more time as it's needed for this test.